.github/workflows/publish-to-pypi.yml

```diff
@@ -1,16 +1,14 @@
-name: Publish to PyPI
-
-on:
-  release:
-    types: [released]
-
-jobs:
-  publish:
-    uses: spacetelescope/action-publish_to_pypi/.github/workflows/workflow.yml@master
-    with:
-      test: false
-      build_platform_wheels: false # Set to true if your package contains a C extension
-    secrets:
-      user: ${{ secrets.PYPI_USERNAME_ASDF_MAINTAINER }}
-      password: ${{ secrets.PYPI_PASSWORD_ASDF_MAINTAINER }} # WARNING: Do not hardcode secret values here! If you want to use a different user or password, you can override this secret by creating one with the same name in your Github repository settings.
-      test_password: ${{ secrets.PYPI_PASSWORD_ASDF_MAINTAINER_TEST }}
+name: Publish to PyPI
+
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
```

Comment:
```suggestion
on:
  release:
    types: [released]
  pull_request:
  workflow_dispatch:
```

Reply:
Thanks!

PRs are currently tested using the OpenAstronomy workflow in the `package` step:
https://github.com/asdf-format/asdf/blob/fca81362afe60f127aea6d25f95790a974e6bf31/.github/workflows/ci.yml#L121-L129

I'll take a look at combining these in a follow-up PR.